cognee-mcp/src/server.py

```
@@ -1,253 +1,164 @@
+                graph_model = load_class(graph_model_file, graph_model_name)
+            else:
+                graph_model = KnowledgeGraph
+
+            await cognee.add(text)
+
+            try:
+                await cognee.cognify(graph_model=graph_model)
+                logger.info("Cognify process finished.")
+            except Exception as e:
+                logger.error("Cognify process failed.")
+                raise ValueError(f"Failed to cognify: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)